src/pages/404.js

```diff
@@ -1,51 +1,22 @@
 import * as React from "react"
 import { Link } from "gatsby"
+import Img from "../images/404.png";
 
-// styles
-const pageStyles = {
-  color: "#232129",
-  padding: "96px",
-  fontFamily: "-apple-system, Roboto, sans-serif, serif",
-}
-const headingStyles = {
-  marginTop: 0,
-  marginBottom: 64,
-  maxWidth: 320,
-}
-
-const paragraphStyles = {
-  marginBottom: 48,
-}
-const codeStyles = {
-  color: "#8A6534",
-  padding: 4,
-  backgroundColor: "#FFF4DB",
-  fontSize: "1.25rem",
-  borderRadius: 4,
-}
-
-// markup
 const NotFoundPage = () => {
   return (
-    <main style={pageStyles}>
+    <main>
       <title>Not found</title>
-      <h1 style={headingStyles}>Page not found</h1>
-      <p style={paragraphStyles}>
-        Sorry{" "}
-        <span role="img" aria-label="Pensive emoji">
-          😔
-        </span>{" "}
-        we couldn’t find what you were looking for.
-        <br />
-        {process.env.NODE_ENV === "development" ? (
-          <>
-            <br />
-            Try creating a page in <code style={codeStyles}>src/pages/</code>.
-            <br />
-          </>
-        ) : null}
-        <br />
-        <Link to="/">Go home</Link>.
+      <h1>Page not found</h1>
+      <p>
+        Oops, looks like the page cannot be found...
```

Comment:
not sure we need this,
404 nowadays is very expressive I think :)

the header takes space and the 'go back' is not visible without scrolling
![image](https://user-images.githubusercontent.com/45601541/122422245-49c64100-cf8d-11eb-87b3-03a872104ab8.png)

Reply:
Now, it's sommething like 
![image](https://user-images.githubusercontent.com/4921914/122530547-dd941d80-d01e-11eb-97c4-efb258c26906.png)